tests/scripts/test-cluster-updates.sh

```
@@ -0,0 +1,65 @@
+}
+
+function finish() {
+  if [ "${LOCAL_MODE:-0}" -eq 1 ]; then
+    rm -rf "$ARTIFACT_DIR"
+  fi
+}
+trap finish EXIT
+
+# ARTIFACT_DIR is set automatically in Prow; fall back to a temp dir locally
+if [ -z "${ARTIFACT_DIR:-}" ]; then
+  export ARTIFACT_DIR=$(mktemp -d)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)